robolectric/src/test/java/org/robolectric/shadows/ShadowConnectivityManagerTest.java

```diff
@@ -147,10 +147,4 @@ public void unregisterCallback_shouldRemoveCallbacks() throws Exception {
     connectivityManager.unregisterNetworkCallback(callback1);
     assertEquals(0, shadowConnectivityManager.getNetworkCallbacks().size());
   }
-
-  @Test(expected=IllegalArgumentException.class) @Config(sdk = 21)
```

Comment:
The unregisterNetworkCallback method now has a version which takes a PendingIntent (http://developer.android.com/reference/android/net/ConnectivityManager.html#unregisterNetworkCallback(android.app.PendingIntent)). This makes passing in a single null argument ambiguous which prevents this from compiling under 23.